lib/fluent/plugin_helper/thread.rb

```diff
@@ -66,6 +66,12 @@ def thread_create(title)
             yield
             thread_exit = true
           rescue Exception => e
+            if @under_plugin_development
+              STDERR.puts "\nError raised in thread from #thread_create, #{e.class}:#{e.message}"
```

Comment:
Yes, this code expects that leading text is '.' (means progress of test cases) from test runner, without tailing newline.